content/06.methods.md

```diff
@@ -588,34 +588,46 @@ EXTEND scores have been further compared, using Spearman rank correlation, betwe
 ### KEY RESOURCES TABLE
 | REAGENT or RESOURCE     | SOURCE                                | IDENTIFIER                                                                                   |                               
 |-----------------|-------------------------------------------------|------------------------------------------------------------------------|
-|Biological samples|||
-|                    |                        |                    |
-|                    |                        |                    |
-|                    |                        |                    |
-|                    |                        |                    |
 |Critical commercial assays|||
-|                    |                        |                    |
-|                    |                        |                    |
-|                    |                        |                    |
-|                    |                        |                    |
+| Recover Cell Culture Freezing media    | Gibco           | 12648010    |
+| Hank's Balanced Salt Solution (HBSS)   | Gibco           | 14175095    |
+| Papain                                 | SciQuest        | LS003124    |
+| Ovomucoid                              | SciQuest        | 542000      |
+| DNase                                  | Sigma           | 10104159001 |
+| 100μm cell strainer                    | Greiner Bio-One | 542000      |
+| DMEM/F-12 medium                       | Sigma           | D8062       |
+| Fetal Bovine Serum (FBS)               | Hyclone         | SH30910.03  |
+| GlutaMAX                               | Gibco           | 35050061    |
+| Penicillin/Streptomycin-Amphotericin B | Lonza           | 17-745E     |
+| Normocin                               | Invivogen       | ant-nr-2    |
+| B-27 supplement minus vitamin A        | Gibco           | 12587-010   |
+| N-2 supplement                         | Gibco           | 17502001    |
+| Epidermal growth factor                | Gibco           | PHG0311L    |
+| Basic fibroblast growth factor         | PeproTech       | 100-18B     |
+| Heparin                                | Sigma           | H3149       |
+| DNA/RNA AllPrep Kit                    | Qiagen          | 80204       |
+| TruSeq RNA Sample Prep Kit             | Illumina        | FC-122-1001 |
+| KAPA Library Preparation Kit           | Roche           | KK8201      |
+| AllPrep DNA/RNA/miRNA Universal kit    | Qiagen          | 80224       |
+| RNase A                                | Qiagen          | 19101       |
+| QIAsymphony DSP DNA Midi Kit           | Qiagen          | 937255      |
+| KAPA HyperPrep kit                     | Roche           | KK8541      |
+| RiboErase kit                          | Roche           | KK8484      |
```

Comment:
Double check `KK8484` - https://rochesequencingstore.com/?s=KK8484&post_type=product&type_aws=true

Reply:
I will have to check with @mkoptyra on the rest of these - I wonder if they used to exist and now they don't or if they are wrong.

Reply:
hmm, looks like roche has a number different from kapa - will have to change

Reply:
updated to 07962304001